items.md

```diff
@@ -227,69 +227,86 @@ the discoverable operation will result in:
   "type": "item"
 ```
 ## Linked entities
-### Bitstreams
 
-**GET /api/core/items/<:uuid>/bitstreams**
+### Bundles
 
-Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4science.it/dspace-spring-rest/api/core/items/1911e8a4-6939-490c-b58b-a5d70f8d91fb/bitstreams>
+**GET /api/core/items/<:uuid>/bundles**
 
-It returns the bitstreams within this item. See the bitstream endpoint for more info](bitstreams.md#Single Bitstream)
+Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4science.it/dspace-spring-rest/api/core/items/1911e8a4-6939-490c-b58b-a5d70f8d91fb/bundles>
 
-The supported parameters are:
-* page, size [see pagination](README.md#Pagination)
+It returns the bundles within this item. See the [bundle endpoint](bundles.md) for more info
 
-**POST /api/core/items/<:uuid>/bitstreams**
-
-Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4science.it/dspace-spring-rest/api/core/items/1911e8a4-6939-490c-b58b-a5d70f8d91fb/bitstreams>
-
-Curl example:
-```
-curl 'https://dspace7.4science.cloud/dspace-spring-rest/api/core/items/2f4ec582-109e-4952-a94a-b7d7615a8c69/bitstreams' \
- -XPOST -H 'Content-Type: multipart/form-data' \
- -H 'Authorization: Bearer eyJhbGciOiJI...' \
- -F "file=@Downloads/test.html" \
- -F 'properties={ "name": "test.html", "metadata": { "dc.description": [ { "value": "example file", "language": null, "authority": null, "confidence": -1, "place": 0 } ]}, "bundleName": "ORIGINAL" };type=application/json'
-```
-
-* The item is determined using the ID in the URL
-* The file is uploaded using multipart/form-data
-* The metadata of the bitstream is included as a json property, and with layout, it looks like:
 ```json
 {
-  "name": "test.html",
-  "metadata": {
-    "dc.description": [
-      {
-        "value": "example file",
-        "language": null,
-        "authority": null,
-        "confidence": -1,
-        "place": 0
+  "bundles" : [
+  {
+    "uuid": "d3599177-0408-403b-9f8d-d300edd79edb",
+    "name": "ORIGINAL",
+    "handle": null,
+    "metadata": {},
+    "type": "bundle",
+    "_links" : {
+      "primarybitstream" : {
+        "href" : "https://dspace7-entities.atmire.com/rest/api/core/bitstreams/ac49f361-4ffd-47a4-8eb2-e6c73c3f3e76"
+      },
+      "orderedbitstreams" : {
```

Comment:
Shouldn't this just be called `bitstreams` to align it with the `/bundles/{uuid}` endpoint?